python/tank/platform/engine.py

```diff
@@ -661,6 +719,29 @@ def created_qt_dialogs(self):
         """
         return self.__created_qt_dialogs
 
+
+    @property
+    def host_info(self):
```

Comment:
I think we should just have `host_name` and `host_version` properties. I'm not sure I see the benefit of having the intermediate object. Let me know if you disagree.